UI/cypress/support/commands.js

```diff
@@ -77,6 +77,20 @@ Cypress.Commands.add("isWithinViewPort", (element) => {
 Cypress.Commands.add("isCorrectJumpToLocation", ($element) => {
   cy.visit($element.attr("href"));
   cy.get('[data-testid="type-of-tuition"]').first().should("not.be.empty");
+  cy.get('[data-testid="pricing-group-size-column"]')
```

Comment:
This feels very wrong in a `isCorrectJumpToLocation` command. Have we overloaded this with some other stuff?

Reply:
The `type-of-tuition` check feels out of place too

Reply:
As agreed - needs a bigger refactor, renamed command so better, for now.